google/resource_container_cluster.go

```diff
@@ -679,22 +709,33 @@ func flattenClusterNodeConfig(c *container.NodeConfig) []map[string]interface{}
 	return config
 }
 
-func flattenClusterNodePools(d *schema.ResourceData, c []*container.NodePool) []map[string]interface{} {
-	count := len(c)
-
-	nodePools := make([]map[string]interface{}, 0, count)
+func flattenClusterNodePools(d *schema.ResourceData, config *Config, c []*container.NodePool) ([]map[string]interface{}, error) {
+	nodePools := make([]map[string]interface{}, 0, len(c))
 
 	for i, np := range c {
+		// Node pools don't expose the current node count in their API, so read the
```

Comment:
Have we talked with the GKE folks about updating their API to include the node count?